data/json/terrain.json

```diff
@@ -9,6 +9,27 @@
       { "item": "splinter", "count": [ 1, 5 ] }
     ]
   },
+  {
+    "type": "terrain",
+    "id": "t_planter",
+    "name": "planter",
+    "symbol": "!",
+    "color": "black",
+    "move_cost": 1,
+    "flags": [ "PLANTABLE" ],
+    "examine_action": "dirtmound"
+  },
+  {
+    "type": "furniture",
+    "id": "f_planter",
+    "name": "planter",
+    "symbol": "!",
+    "color": "black",
+    "move_cost_mod": 1,
+    "flags": [ "PLANTABLE" ],
+    "examine_action": "dirtmound",
+    "required_str": 1
```

Comment:
Should these have bash results when hit? I'd imagine you'd get either metal or wood parts, with maybe nails and soil?

Reply:
Yeah, they'll be like plants, where you can bash them into planters, and then bash the planter for diminished return for whatever it's construction recipe is (probably nails/wood/soil).

Reply:
Or not, because there's not equivalent of `set_ter` for furniture bashing.